sensor_msgs/PointCloud2 cloud
string target_frame
---
sensor_msgs/PointCloud2 cloud

